apis/r/R/TileDBArray.R

```diff
@@ -289,17 +312,17 @@ TileDBArray <- R6::R6Class(
 
   private = list(
 
-    # Reopen the array in a different mode
-    reopen = function(mode = c("READ", "WRITE")) {
-      mode <- match.arg(mode)
-      if (private$.mode == mode) {
-        return(invisible(self))
-      }
-      self$close()
-      invisible(
-        self$open(mode = mode, internal_use_only = "allowed_use")
-      )
-    },
+    # # Reopen the array in a different mode
+    # reopen = function(mode = c("READ", "WRITE")) {
+    #   mode <- match.arg(mode)
+    #   if (private$.mode == mode) {
+    #     return(invisible(self))
+    #   }
+    #   self$close()
+    #   invisible(
+    #     self$open(mode = mode, internal_use_only = "allowed_use")
+    #   )
+    # },
```

Comment:
We tend to remove commented-out code.

Reply:
🤦 One day, I'll learn to read-through my PRs and remove unused code. Thanks for catching!